framework/console/Controller.php

```diff
@@ -697,7 +697,7 @@ protected function getActionMethodReflection($action)
 
     /**
      * Parses the comment block into tags.
-     * @param \Reflector $reflection the comment block
+     * @param \ReflectionClass|\ReflectionProperty|\ReflectionFunctionAbstract $reflection the comment block
```

Comment:
`\ReflectionMethod|\ReflectionProperty`

Reply:
See https://github.com/yiisoft/yii2/pull/19431#discussion_r896194416 regarding `ReflectionFunctionAbstract`.

Fine to remove `ReflectionClass`, but it seemed obvious that it should be included since `getDocComment()` is also defined there, so why not?